dist/src/main/java/io/camunda/zeebe/shared/management/ClusterEndpoint.java

```diff
@@ -147,7 +157,7 @@ private ResponseEntity<?> cancelChange(final String changeId) {
     } catch (final NumberFormatException ignore) {
       return invalidRequest("Change id must be a number");
     } catch (final Exception error) {
-      return mapError(error);
+      return mapError(error.getCause());
```

Comment:
:thought_balloon: Same here: what if there is no cause?
:wrench: I think "unwrapping" the exception should be handled in `mapError`. For `CompletionException`, use the cause, for others use the exception directly.